config/webpack/webpack.config.js

```diff
@@ -28,6 +30,7 @@ const webpackConfig = (options, env, argv) => {
   const filename = argv['output-filename'] || fileNameStategy;
   const outputPath = argv['output-path'] || path.join(rootPath, 'build');
   const publicPath = argv['output-public-path'] || '';
+  const themeConfig = defaultTheme ? { theme: defaultTheme } : undefined;
```

Comment:
If this filters out unspecified themes, how does not include all the themes when packing production assets? Is this configuration specifically for testing?

Reply:
When programmatically setting the default theme you only get the default theme. I see it as specifically for testing, yes.